packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -207,6 +207,17 @@ export function UserListTable() {
           );
         },
         filterFn: (rows, id, filterValue) => {
+          if (filterValue.length === 1 && filterValue[0] === "PENDING") {
+            // Show only pending members
+            return !rows.original.accepted;
+          }
+
+          if (filterValue.length > 1 && filterValue.includes("PENDING")) {
+            // Show pending members and the selected roles
+            return !rows.original.accepted && filterValue.includes(rows.getValue(id));
```

Comment:
I think you meant to use OR here instead of &&

Reply:
I thought showing role AND pending would be better

i.e if you check admin and pending boxes, you show admins that are pending

Reply:
but all other checkboxes doesn't work like this. I also tried selection owner and members and it returned me both members and owner.

Reply:
I think OR is fine for now. I'll change it later if we don't need it.